pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>com.ghost-inspector.jenkins</groupId>
   <artifactId>ghost-inspector</artifactId>
-  <version>3.2-SNAPSHOT</version>
+  <version>3.3</version>
```

Comment:
I can't remember offhand, but I think the plugin publishing process might increment this for us... I'll check.